utils/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -136,7 +136,18 @@ function handleError(context: types.IActionContext, callbackId: string, error: u
         } else {
             context.telemetry.properties.result = 'Failed';
             context.telemetry.properties.error = errorData.errorType;
+
+            /**
+             * @param errorMessage
+             * @deprecated
+             * Continue to emit telemetry for clients who are still using this property. You should suppress this property if you need to migrate to the new replacement.
+             *
+             * @param errorMessage2
+             * A duplicate replacement of the `errorMessage` telemetry property which should be used instead.
+             */
             context.telemetry.properties.errorMessage = errorData.message;
+            context.telemetry.properties.errorMessage2 = errorData.message;
```

Comment:
Numbers in property names get censored in telemetry (don't ask my why, they just do). Not sure of a good solution though.

Reply:
At least in the GDPR catalog.

Reply:
I'll take a look for sure tomorrow. Please hold on merging until then. I have an example from Docker I can look at.